packages/grafana-schema/src/common/mudball.cue

```diff
@@ -203,7 +203,7 @@ FieldTextAlignment: "auto" | "left" | "right" | "center" @cuetsy(kind="type")
 
 // Internally, this is the "type" of cell that's being displayed
 // in the table such as colored text, JSON, gauge, etc.
-// The color-background-solid, gradient-gauge, and lcd-gauge 
+// The color-background-solid, gradient-gauge, and lcd-gauge
 // modes are deprecated in favor of new cell subOptions
 TableCellDisplayMode: "auto" | "color-text" | "color-background" | "color-background-solid" | "gradient-gauge" | "lcd-gauge" | "json-view" | "basic" | "image" | "gauge" @cuetsy(kind="enum",memberNames="Auto|ColorText|ColorBackground|ColorBackgroundSolid|GradientGauge|LcdGauge|JSONView|BasicGauge|Image|Gauge")
```

Comment:
Maybe this should also be in the `table.cue`?

Reply:
oh yeah probably